controllers/DefaultController.php

```diff
@@ -263,17 +263,21 @@ public function actionRegister()
                             Yii::$app->session->setFlash('error', Yii::t('usr', 'Failed to send an email.').' '.Yii::t('usr', 'Try again or contact the site administrator.'));
                         }
                     }
-                    if ($model->getIdentity()->isActive()) {
-                        if ($model->login()) {
-                            return $this->afterLogin();
+                    if ($model->getIdentity() instanceof \nineinchnick\usr\components\ActivatedIdentityInterface) {
```

Comment:
If the identity is not _activable_, shouldn't we assume the new account is active and try to log in instead of just redirecting without _any_ message to the user? That's a bit confusing.